src/rust/graph-merger/src/service.rs

```diff
@@ -55,26 +78,23 @@ impl From<&GraphMergerError> for kafka::StreamProcessorError {
 
 #[derive(Clone)]
 pub struct GraphMerger {
-    mg_client: Arc<DgraphClient>,
-    reverse_edge_resolver: ReverseEdgeResolver,
+    mg_client: GraphMutationClient,
```

Comment:
`mg_client` meaning `master_graph_client` is so outdated i'm gonna rename this >:)